src/nidcpower/metadata/attributes.py

```diff
@@ -209,7 +209,7 @@
         'access': 'read-write',
         'documentation': {
             'description': '\nSpecifies whether the measurement returned from any measurement call starts with a new measurement call (VI_TRUE) or returns a measurement that has already begun or completed(VI_FALSE).\nfor information about supported devices.\nWhen you set the NIDCPOWER_ATTR_SAMPLES_TO_AVERAGE attribute in the Running state, the output channel measurements might move out of synchronization. While NI-DCPower automatically synchronizes measurements upon the initialization of a session, you can force a synchronization in the running state before you run the niDCPower_MeasureMultiple function. To force a synchronization in the running state, set this attribute to VI_TRUE, and then run the niDCPower_MeasureMultiple function, specifying all channels in the channel name parameter. You can set the NIDCPOWER_ATTR_RESET_AVERAGE_BEFORE_MEASUREMENT attribute to VI_FALSE after the niDCPower_MeasureMultiple function completes.\nDefault Value: VI_TRUE\n',
-            'note': 'This attribute is not supported by all devices. Refer to Supported Attributes by Device topic'
+            'note': '\nThis attribute is not supported by all devices. For more information about supported attributes, search ni.com for Supported Attributes by Device.\n'
```

Comment:
```suggestion
            'note': '\nThis attribute is not supported on all devices. For more information about supported devices, search ni.com for Supported Attributes by Device.\n'
```

The above suggestion will make it consistent with what we have for methods - `This function is not supported on all devices. For more information about supported devices, search ni.com for Supported Functions by Device.`

Please update internal metadata to reflect this change.

Reply:
I've updated the internal metadata and this PR with the latest rounds of updates to the docs